modules/force_rtl.py

```diff
@@ -0,0 +1,50 @@
+"""
+Forces RTL (return to landing)
```

Comment:
I believe RTL stands for return to launch (so it goes back to the takeoff point).
Can we phrase is like this `Forces drone to return to launch (RTL)`